packages/shared/api/presentation-shared.api.md

```diff
@@ -281,6 +281,8 @@ export interface ECSqlQueryExecutor {
 
 // @beta
 export interface ECSqlQueryReaderOptions {
+    // (undocumented)
+    restartToken?: string;
```

Comment:
Need a changelog for this and the change in `core-interop`

Reply:
Maybe it is enough to add changelog only to `shared`? `core-interop` should get `Updated dependencies:
@itwin/presentation-shared@0.2.0`

Reply:
Hm, I'm not totally sure... IMO, a dependency update means you made a `package.json` change, but doesn't mean you actually changed anything else. So I think that having an entry, saying that you're now making use of the new feature that came with the dependency update, makes sense.